derive_builder/tests/compiletests.rs

```diff
@@ -18,7 +18,7 @@ fn run_mode(mode: &'static str) {
         panic!("Directory does not exist: {:?}", test_dir);
     }
 
-    let mut config = compiletest::default_config();
+    let mut config = compiletest::Config::default();
```

Comment:
compiletest removed function `default_config`, as discussed in [their docs](https://docs.rs/compiletest_rs/0.2.10/compiletest_rs/fn.default_config.html):

    Deprecated since 0.2.9: Use Config::default() instead. This method will be removed in version 0.3.0